cli/src/cli_util.rs

```diff
@@ -1394,7 +1394,7 @@ See https://github.com/martinvonz/jj/blob/main/docs/working-copy.md#stale-workin
         {
             writeln!(
                 ui.warning(),
-                r#"Name and email not configured. Until configured, your commits will be created with the empty identity, and can't be pushed to remotes. To configure, add something like the following to $HOME/.jjconfig.toml:
+                r#"Name and email not configured. Until configured, your commits will be created with the empty identity, and can't be pushed to remotes. To configure, add something like the following to your config file:
   user.name = "Some One"
   user.email = "someone@example.com""#
```

Comment:
Should we instead use `jj config set --user` here too?

```
Name and email not configured. Until configured, your commits will be created with the empty identity, and can't be pushed to remotes. To configure, run something like the following:
jj config set --user user.name = "Some One"
jj config set --user user.email = "someone@example.com"
```

Reply:
Ooh, I didn't know we had that! That's the best of both worlds.

Reply:
Done.